site/content/en/latest/concepts/client-traffic-policy.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "ClientTrafficPolicy"
+---
+## Before you Begin
+- [Gateway API Extensions](gateway-api-extensions.md)
+
+## Overview
+
+`ClientTrafficPolicy` is an extension to the Kubernetes Gateway API that controls how Envoy Gateway handles incoming traffic from clients. It can configure connection behavior, security settings, and protocol optimizations at the gateway level.
```

Comment:
lets rm security settings from here

Reply:
instead of protocol optimizations, can we say protocol specific configuration

Reply:
my bad things, there's a section on tls in there, can we explicitly say `TLS` settings instead